working-with-openai/open-ai-api-models.md

```diff
@@ -1,76 +1,84 @@
+| Description | Use Case |
+| ----------- | -------- | 
+| Natural Language Processing, which can be used to understand and process natural language like free form text or a document in order to generate code or respond to questions. | Chatbots, content generation |
+| Generating or editing an image or images via a Natural Language prompt. | Image generation, Image Manipulation |
+| Converting text to natural sounding speech.| Text to speech conversion | 
+| Converting audio to text or analyzing an audio clip guided by a text prompt. | Audio transcription, Audio descriptions, Captioning | 
+| Analyzing a video, image, or images guided by a text prompt. | Image or video descriptions, Image content analysis |
 
-If you are interested in learning about other models that are no longer widely used, OpenAI provides a list [here](https://platform.openai.com/docs/deprecations)  
+As an example, the original model upon which OpenAI was built was GPT-3. GPT stands for Generative Pre-Trained Transformer and GPT-3 was trained on 175 billion parameters. GPT-3 has now been deprecated in favor of even larger models known as GPT-3.5 and GPT-4. 
+
+When choosing a model to work with we can start by asking ourselves, what kind of output do we need? Are we looking for text, audio, images, etc? That will narrow down which API providers could meet our needs. At that point we need to look at each model's limitations and pricing to see what fits both our budget and the kind of traffic we think we'll need to support for our feature. Most free tier limits are great for testing things out, but not scalable as a permanent solution if an AI feature becomes popular.  
 
 ### !callout-info
 
-## Where are OpenAI Models Used? 
-As we will see in the next lesson on endpoints, these models are most commonly used as query parameters within our API requests to tell OpenAI which model we would like to use when processing a specific request. 
+## Where is an AI Model Used? 
+These models are most commonly used as query parameters within our API requests to tell the AI API which model we would like to access when processing a specific request. 
 
 ### !end-callout
-
-### !callout-info
+ 
+## Google Gemini 1.5 Flash
   
-## OpenAI Models and Pricing
-  
-The OpenAI API is not a free service at this point in time, so each model has a different pricing structure that depends on the size of the request that is being made. Pricing information can be found [here](https://openai.com/api/pricing)  
+While we work with the Google Gemini API, we will be using the Gemini Flash 1.5 model specifically. We are using this model because it's free tier fits our needs to show off some text-based use cases, while allowing for exploration past what we will cover in class for folks who are interested. We will cover facts we think are useful below, for more information, check out the [Google Gemini API Documentation](https://ai.google.dev/gemini-api/docs/models/gemini).
```

Comment:
```suggestion
While we work with the Google Gemini API, we will be using the Gemini Flash 1.5 model specifically. We are using this model because it's free tier fits our needs to show off some text-based request and response use cases, while allowing for exploration past what we will cover in class for folks who are interested. We will cover facts we think are useful below, for more information, check out the [Google Gemini API Documentation](https://ai.google.dev/gemini-api/docs/models/gemini).
```